odoo/signals.py

```
@@ -0,0 +1,123 @@
+
+@receiver(post_save, sender=User)
+def sync_user_to_odoo(sender, instance, created, **kwargs):
+    """
+    Signal handler to sync user to Odoo when created or updated.
+    """
+    odoo_user = OdooUserResource()
+    odoo_user.sync_user_to_odoo_api(instance)
+
+
+@receiver(post_save, sender=Invoice)
+def save_fields_before_update(sender, instance, raw, using, update_fields, **kwargs):
```

Ruff (0.14.3)

Unused function argument: `sender`

(ARG001)